### PR TITLE
Changed names and descriptions of armor CBMs, increased protection

### LINKED ISSUE #45735: Bionic armors provide more protections & more encumbrance
# Is your feature request related to a problem? Please describe.
Bionic armors now provide 3 bash_protec, 3 cut_protec and 3 bullet_protec, which is directly inherited from Cataclysm and never changed since, while other statistics about melee attack have changed a lot. This cause a funny situation that these **sci-fi alloy plating armor** provides protections same as **paper armors** do, and in real game-play experience players rarely install these CBMs to their characters. 

# Describe the solution you'd like
Let bionic armors (XXX Alloy Plating CBM) provide more protections whilst cause more encumbrance(in order to rebalance game data).

# Describe alternatives you've considered
No alternatives yet...

# Additional context
Some pics.
image
Paper armor

image
Alloy plating CBM

### PR DESCRIPTION
**Pull request by Kelenius**
#### Summary

SUMMARY: Balance "Slightly improved and refluffed armor CBMs"

#### Purpose of change

Resolves #45735
Alternative to #45755

#### Describe the solution

I don't like the changes proposed in #45755 and here's an alternative option.

I renamed and changed the description of these CBMs to explain why the armor provided is so low and encumbrance-free - it's a thin layer of alloy armor implanted under the skin (which is how I imagined them previously). I increased armor from 3 to 4, because 3 is indeed too low. I added a mention of this to Carbon Filament, to explain that while both of them are subdermal armor, they're intended to be able to work together. I also added mouth armor to both head armor and carbon filament, which they didn't have before and it seems to be an oversight; head armor even says it protects `head and jaw`.

#### Testing

Spawned a fresh bionic cop and spawned items to check that it's OK.

### CODE REVIEW

**Comment by Fris0uman** [comment]
#45755  improves the code and expands CBM implementation, it's way more interesting than changing values and descriptions

**Comment by Kelenius** [comment]
Improvements can be implemented separately. Please focus on the important parts.

**Comment by emptytriangle** [comment]
the major things at issue (there are some other minor discussions) in #45755 are the armor and encumbrance values. Since it improves the code and expands CBM implementation it's better to move that discussion elsewhere. It would be best to drop the armor and encumbrance value changes from that PR and do them separately

File: data/json/bionics.json [hunk 1]
```diff
@@ -137,13 +137,13 @@
...
-    "cut_protec": [ [ "head", 3 ], [ "mouth", 3 ] ],
-    "bullet_protec": [ [ "head", 3 ], [ "mouth", 3 ] ],
-    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ],
+    "name": { "str": "Subdermal Armor - Head" },
+    "description": "A thin layer of alloy armor is implanted under your skin, protecting both your head and jaw regions.",
+    "occupied_bodyparts": [ [ "head", 5 ], [ "mouth", 1 ] ],
+    "bash_protec": [ [ "head", 4 ], [ "mouth", 4 ] ],
+    "cut_protec": [ [ "head", 4 ], [ "mouth", 4 ] ],
+    "bullet_protec": [ [ "head", 4 ], [ "mouth", 4 ] ],
+    "flag": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ],
```
Comment (ZhilkinSerg) [inline data/json/bionics.json:146]:
```suggestion
    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ],
```

**Comment by kevingranade** [comment]
obsoleted by #51578

**Pull request closed without merging**